<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_admin_site</class>
 <widget class="QWidget" name="show_admin_site">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>景点名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_order">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>总预订量：1000张</string>
   </property>
  </widget>
  <widget class="QLabel" name="area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>地区</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_low">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>淡季票价:1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="price_high">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>旺季票价:1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="profit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>利润:1000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="ma_price">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>维护费用:1000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="all_num">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>85</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>总访问人数:10000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="autumn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>秋季人数:1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="winter">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>冬季人数:1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="summer">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>夏季人数:1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="spring">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>春季人数:1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="more_info">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄分布</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
